Extension/package.json

```diff
@@ -3050,6 +3050,22 @@
                         "default": false,
                         "markdownDescription": "%c_cpp.configuration.addNodeAddonIncludePaths.markdownDescription%",
                         "scope": "application"
+                    },
+                    "C_Cpp.includeHeader": {
```

Comment:
Right now this is in the "Miscellaneous" sub-heading.  I think it should go in the `"title": "%c_cpp.subheaders.intelliSense.title%"` object unless we plan to add a refactoring group.

Should this be renamed to `C_Cpp.refactor.includeHeader` to scope it for refactorings?

Reply:
Scoped to refactor in case other refactoring settings are added in the future. Also, to clarify this only applies to refactoring operations.